include/commissioning/commissioner.h

```diff
@@ -50,14 +50,13 @@ extern "C" {
  * This function enables the Thread Commissioner role.
  *
  * @param[in]  aInstance         A pointer to an OpenThread instance.
- * @param[in]  aPSKd             A pointer to the PSKd.
  * @param[in]  aProvisioningUrl  A pointer to the Provisioning URL (may be NULL).
  *
  * @retval kThreadError_None         Successfully started the Commissioner role.
  * @retval kThreadError_InvalidArgs  @p aPSKd or @p aProvisioningUrl is invalid.
  *
  */
-ThreadError otCommissionerStart(otInstance *aInstance, const char *aPSKd, const char *aProvisioningUrl);
+ThreadError otCommissionerStart(otInstance *aInstance, const char *aProvisioningUrl);
```

Comment:
Harness THCI API has a param for setting PSKc when enabling commissioner.
The API is allowCommissioner(PSKc).

Also it has a separate API for setting Provisioning URL.